src/__test__/components/UserTable.test.tsx

```diff
@@ -16,18 +17,22 @@ describe('the UserTable component', () => {
   const tableHeaders: string[] = ['Column1', 'Column2', 'Column3'];
 
   it('should have the correct headers', () => {
+    /*
     const { getByTestId } = render(
```

Comment:
Remove this code block and rely on git for history

Reply:
Done!